chia/util/streamable.py

```diff
@@ -262,7 +267,11 @@ def stream_one_item(self, f_type: Type, item, f: BinaryIO) -> None:
             raise NotImplementedError(f"can't stream {item}, {f_type}")
 
     def stream(self, f: BinaryIO) -> None:
-        for f_name, f_type in get_type_hints(self).items():  # type: ignore
+        try:
+            fields = self.__annotations__  # pylint: disable=no-member
+        except Exception:
```

Comment:
Might be better to catch the specific exception type. Personally, I kind of prefer `getattr(self, "__annotations__", {})` as being a bit more terse & explicit (even if it's a bit weird-looking).

Reply:
It's too slow